index.html

```diff
@@ -36,6 +36,40 @@ <h1>Crafted for Business</h1>
         <img src="assets/img/slider/2.png" />
         <span class="crafted__right-arrow">&RightArrow;</span>
       </section>
+
+       <section class="services">
+      <div>
+        <h1>Our Services</h1>
+        <p>Stop wasting time and money designing and managing a website that doesn't get results. Happiness guaranteed</p>
+      </div>
+
+      <div>
+      
+        <div>
+          <h2>Graphics Design</h2>
+          <img src="/assets/img/f-shape-1.svg" alt="graphic design">
+          <p>Short description for the ones who look for something new. Short description for the ones who look for something new.</p>
+          <a href="#">learn more</a>
+        </div>
+      
+      
+        <div>
+          <h2>Website Design</h2>
+          <img src="/assets/img/f-shape-1.svg" alt="website design">
+          <p>Short description for the ones who look for something new. Short description for the ones who look for something new.</p>
+          <a href="#">learn more</a>
+        </div>
+      
+      
```

Comment:
remove double newline → single newline
will add a prettier config to our repo later so we can keep consistent formatting